.env.example

```diff
@@ -1,37 +0,0 @@
-APP_NAME=Laravel
```

Comment:
Why are you deleting the sample env file?

Reply:
Thought it won't be used, since i have the .env itself.